### PR TITLE
Add Dockerfile for build process

### PR DESCRIPTION
**Pull request by wjxiz1992**
Signed-off-by: Allen Xu <allxu@nvidia.com>

close: #103

### CODE REVIEW

**Review by GaryShen2008 — DISMISSED** [review]

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,85 @@
+###
```
Comment (tgravescs) [inline docker/Dockerfile:1]: add copyright

File: docker/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Build the container
+
+```bash
+docker build -t rapids-ml:latest -f Dockerfile .
+```
+Please check the [Dockerfile](./Dockerfile) for more configurable build arguments.
+
+## Build the project in the container
+
```
Comment (tgravescs) [inline docker/README.md:13]: it might be nice to show the command to enter the docker image even if its just exec bash

File: docker/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+# Config JAVA_HOME
+ENV JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk-amd64
+
+# Donwload maven
+ARG MAVEN_VERSION=3.8.7
+RUN cd /usr/local/ && wget -q https://dlcdn.apache.org/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.zip && \
+     unzip apache-maven-$MAVEN_VERSION-bin.zip && rm -rf apache-maven-$MAVEN_VERSION-bin.zip
+ENV PATH /usr/local/apache-maven-$MAVEN_VERSION/bin:$PATH
+
+
```
Comment (tgravescs) [inline docker/Dockerfile:61]: nit remove extra new line

File: docker/Dockerfile (hunk 1 above)
Comment (wjxiz1992) [reply docker/Dockerfile:18]: Added.

File: docker/README.md (hunk 2 above)
Comment (wjxiz1992) [reply docker/README.md:13]: Added.

**Review by wjxiz1992 — COMMENTED** [review]
Updated according to review comments.

File: docker/Dockerfile (hunk 3 above)
Comment (wjxiz1992) [reply docker/Dockerfile]: removed.

**Review by tgravescs — APPROVED** [review]

**Review by GaryShen2008 — APPROVED** [review]

**Pull request merged**